### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by jburel**
Add more python code check
and exclude directories generating when prep for pypi upload
Check that travis is green and that gallery is still working

### CODE REVIEW

File: omero_gallery/views.py [hunk 1]
```diff
@@ -52,28 +52,28 @@ def index(request, conn=None, **kwargs):
 
 @login_required()
 @render_response()
-def show_group(request, groupId, conn=None, **kwargs):
-    conn.SERVICE_OPTS.setOmeroGroup(groupId)
+def show_group(request, group_id, conn=None, **kwargs):
```
Comment (aleksandra-tarkowska) [inline omero_gallery/views.py]:
```
TypeError: show_group() takes at least 2 arguments (2 given)
```

**Comment by jburel** [comment]
@aleksandra-tarkowska did you have a chance to look at the changes?

**Comment by aleksandra-tarkowska** [comment]
I am afraid I cannot test it because of 

```
TypeError: _getQueryString() takes exactly 2 arguments (1 given)
```

**Comment by will-moore** [comment]
@aleksandra-tarkowska That _getQueryString error is coming from changes in https://github.com/openmicroscopy/openmicroscopy/pull/4950

**Comment by aleksandra-tarkowska** [comment]
thanks @jburel for reminding, looks good

I just noticed few issues with 

```
GET /webtrial/gallery/show_image/8751/ 200 OK
GET jquery.mousewheel.js 404 Not Found
GET jquery.mousewheel.js 404 Not Found
 ```

but these are invalid names, not related to the PR. Noted in https://trello.com/c/cyoBblgp/200-renaming-of-web-app

**Comment by jburel** [comment]
Thanks @aleksandra-tarkowska for the review

**Pull request merged**